cf-nodejs-logging-support-core/log-core.js

```diff
@@ -591,29 +591,28 @@ var writeCustomFields = function (logObject, logger, additionalFields) {
     var providedFields = Object.assign({}, extractCustomFieldsFromLogger(logger), additionalFields);
 
     var customFields = {};
+    var value;
     for (var key in providedFields) {
-        // Skip unregistered fields
+        value = providedFields[key];
 
-        var value = providedFields[key];
-
-        // Write value to customFields object. Stringify, if necessary.
+        // Stringify, if necessary.
         if ((typeof value) != "string") {
             value = stringifySafe(value);
         }
 
-        if(registeredCustomFields.includes(key) && defaultCustomEnabled) {
+        if(defaultCustomEnabled)
             logObject[key] = value;
-        }
-        if (registeredCustomFields.includes(key) && cfCustomEnabled) {
+
+        if (cfCustomEnabled)
```

Comment:
Why was the check for registered custom field name dropped?

Reply:
It is unnecessary in this place, because it is checked again later when the fields are written in the correct order, and it is needed there for function.